runtime.md

```diff
@@ -13,6 +13,13 @@ The state of a container MUST include, at least, the following properties:
 * **`id`**: (string) is the container's ID.
 This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
+* **`status`**: (string) is the runtime state of the container.
+The value MAY be one of:
+  * `created` : the container has been created but the user-specified process has not yet been executed
+  * `running` : the container has been created and the user-specified process is running
```

Comment:
I'd rather have this be “the container process has executed user-specified code has not exited”.  In Linux, in any case “running” is a technical term for one possible process state, and the status you're shooting for here also covers “sleeping”, “waiting”, and “stopped”.